src/Mono.Android/Java.Util/IList.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using Android.Runtime;
+
+namespace Java.Util;
+
+#if ANDROID_35
+public partial interface IList
+{
+	// This gets generated as:
```

Comment:
I don't understand this comment.  Do you mean the body of `n_Reversed()`?  If so, should the comment be moved there?

Reply:
Moved.